src/bin/main.rs

```diff
@@ -568,6 +573,19 @@ impl GetOptsOptions {
             }
         }
 
+        if matches.opt_present("recursive") {
+            if let Some(skip) = options.skip_children {
```

Comment:
I think this can be simplified and so the additional `if` below could be removed:

```rust
            if let Some(true) = options.skip_children {
```

Reply:
Good spot, will update!